js/analyses/checkForKeywordDoubles.js

```diff
@@ -0,0 +1,21 @@
+/** @module analyses/checkForKeywordDoubles */
+
+/**
+ * Checks the keyword in an array of used keywords. If the keyword is in this array, it will return the
+ * number of times the keyword is found, and an ID if it was used once before.
+ *
+ * @param {string} keyword The keyword to check in the array.
+ * @param {array} usedKeywords The array with used keywords and IDs.
+ */
+module.exports = function( keyword, usedKeywords ) {
+	var result = { count: 0, id: 0 };
+	if ( typeof keyword !== "undefined" ) {
+		if ( typeof usedKeywords !== "undefined" && typeof usedKeywords[ keyword ] !== "undefined" ) {
```

Comment:
We have decided to remove checks for `undefined` in the module functions, so these should go as well.